cockroachcloud/frequently-asked-questions.md

```diff
@@ -17,7 +17,26 @@ This page answers the frequently asked questions about {{ site.data.products.ser
 
 ### In what clouds and regions is {{ site.data.products.db }} available?
 
-The  {{ site.data.products.db }} Console will always show the latest available cloud infrastructure providers and regions. Open the [**Create Cluster**](https://cockroachlabs.cloud/cluster/create) page and select a Plan to see which providers and regions are currently available.
+The following regions are available for {{ site.data.products.dedicated }}:
+
+GCP                                          | AWS
+---------------------------------------------|------
+Frankfurt, Germany (`europe-west3`)          | Osaka, Japan (`ap-northeast-3`)
```

Comment:
Strongly suggest alphabetizing each of these lists for the readers' ease of finding their desired region

My sense is it's better to put the region name first and alphabetize by that e.g. `us-west1` (Oregon) but idk 

(unless there is some reason not to alphabetize I may not be aware of)